<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EvokedPotentialDialog</class>
 <widget class="QDialog" name="EvokedPotentialDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>173</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>EvokedPotentialDialog</string>
  </property>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>331</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>RFA</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>M1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>DMS</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>DLS</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>GP</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Thal</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>STN</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>SNr</string>
    </property>
   </item>
  </widget>
  <widget class="QSpinBox" name="stimchannelbox">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>60</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>8</number>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>60</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Main Stimulation Channel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="calculateButton">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>130</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Calculate EP</string>
   </property>
  </widget>
  <widget class="QPushButton" name="quitButton">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>130</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="leftradio">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>82</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Left</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="rightradio">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>10</y>
     <width>82</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Right</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="seg1">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>70</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="seg2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>100</y>
     <width>70</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>2</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="seg3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>120</y>
     <width>70</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>3</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="seg4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>140</y>
     <width>70</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>4</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Segment</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
